train/NEAT/config-feedforward

```diff
@@ -0,0 +1,73 @@
+[NEAT]
+fitness_criterion       = max
+fitness_threshold       = 1e10
+no_fitness_termination  = true
+pop_size                = 150
+reset_on_extinction     = true 
+
+[DefaultGenome]
+num_inputs              = 208
+num_hidden              = 0
+num_outputs             = 6
+
+activation_default      = random
+activation_mutate_rate  = 0.05
+activation_options      = sigmoid
+
+aggregation_default     = random
+aggregation_mutate_rate = 0.0
+aggregation_options     = sum
+
+bias_init_mean          = 0.0
+bias_init_stdev         = 1.0
+bias_init_type          = uniform
+bias_max_value          = 30.0
+bias_min_value          = -30.0
+bias_mutate_power       = 2.093
+bias_mutate_rate        = 0.7
+bias_replace_rate       = 0.5
+
+compatibility_threshold             = 3.0
+compatibility_disjoint_coefficient  = 2.0
+compatibility_weight_coefficient    = 0.5
+
+conn_add_prob           = 0.2
+conn_delete_prob        = 0.2
+
+feed_forward            = true
+initial_connection      = full_nodirect
+
+node_add_prob           = 0.2
+node_delete_prob        = 0.2
+
+response_init_mean      = 1.0
+response_init_stdev     = 0.1
+response_max_value      = 30.0
+response_min_value      = -30.0
+response_mutate_power   = 0.1
+response_mutate_rate    = 0.1
+response_replace_rate   = 0.1
+
+weight_max_value        = 30
+weight_min_value        = -30
+weight_init_mean        = 0.0
+weight_init_stdev       = 1.0
+weight_mutate_rate      = 0.8
+weight_replace_rate     = 0.1
+weight_mutate_power     = 0.5
+
+enabled_default         = True
+enabled_mutate_rate     = 0.01
+
+[DefaultSpeciesSet]
+compatibility_threshold = 3.0
+
+[DefaultStagnation]
+species_fitness_func    = mean
+max_stagnation          = 15 
+species_elitism         = 3
+
+[DefaultReproduction]
+elitism                 = 2
+survival_threshold      = 0.1
+min_species_size        = 2
```

Comment:
Nit missing newline

Reply:
New line here? following the convention of the people who created neat-python.. they don't have new lines there??

Reply:
Ah NVM, i will put your new line at the end of the file. i thought you wanted it after [DefaultReproduction] haha